src/Microsoft.CSharp/tests/Microsoft.CSharp.Tests.csproj

```diff
@@ -12,7 +12,7 @@
     <Compile Include="CSharpArgumentInfoTests.cs" />
   </ItemGroup>
   <ItemGroup>
-    <ProjectReference Include="..\src\Microsoft.CSharp.csproj">
+    <ProjectReference Include="..\pkg\Microsoft.CSharp.pkgproj">
```

Comment:
Building in VS works fine for me, at least for System.Numerics.Vectors and System.Runtime. You might need the 3.5-beta version of the nuget plugin; not sure.

The tests do not work right away because of this issue: https://github.com/dotnet/buildtools/issues/700, but they work as long as you prep the folder beforehand as the linked issue describes. You also need to point the Start Program to CoreRun's full path, apparently that also got broken at the same time.